src/main/java/frc/robot/RobotContainer.java

```diff
@@ -80,8 +59,8 @@
         private CanalSubsystem canal = new CanalSubsystem();
         private Climber climber = new Climber();
         private Limelight limelight = new Limelight();
-        private NetworkTablesBase networkTables = new NetworkTablesBase();
         private SUB_LED LED = new SUB_LED();
+        private NetworkTablesBase networkTables = new NetworkTablesBase();
```

Comment:
Is `NetworkTablesBase` actually doing anything? Why was a new line added, instead of uncommenting the line that was already there but commented?